<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Predvajalnik</class>
 <widget class="QWidget" name="Predvajalnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Predvajalnik</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="Phonon::VideoPlayer" name="videoPlayer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="Phonon::VolumeSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>450</y>
     <width>109</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="seekSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
